xml/2018-07-15-end-of-year-survey.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<email title="2018 Member Survey"
+  agenda="false"
+  social="false"
+  logos="false">
+
+  <text>
+    This has been a fantastic year for DoCSoc and we hope that you enjoyed all
+    the opportunities we had to provide! With the academic year and (hopefully
+    positive!) examination results out of the way, it's time to kick back and
+    enjoy a well-deserved summer holiday.
+
+    In order to give you even better events and opportunities next year, we
+    would like to ask you to complete a short survey. In case you aren't convinced,
+    we will also give a **£50 Amazon voucher** to three lucky participants. More
+    details below.
+  </text>
+
+  <hr/>
+
+  <image src="https://i.imgur.com/yGnTpVI.jpg" />
+  <text>
+    **WIN ONE OF THREE £50 AMAZON VOUCHERS**
+
+    We’re currently in the process of writing next year’s sponsorship proposal
+    and in the interest of increasing our sponsorship opportunities to benefit
+    all of us, we’d like to collect some data about our performance this year.
+
+    We're also interested in finding out how your experience has been so far
+    with DoCSoc so that we can continue to improve our society.
+
+    The survey is ANONYMOUS. If you wish to be eligible for the prize, we will
+    ask for your username. However, we will NEVER use your username in
+    conjunction with any of the other answers from this survey.
+
+    Complete the survey here: [https://docsoc.co.uk/survey/](https://docsoc.co.uk/survey/)
```

Comment:
The preprocessor does link detection so you shouldn't need to write the link explicitly if you want it to show up with the same text as the destination.